cppwinrt/cmd_reader.h

```diff
@@ -520,18 +493,21 @@ namespace cppwinrt
 
         auto files(std::string_view const& name) const
         {
-            return files(name, [](auto&&) {return true; });
```

Comment:
Not sure I love this change. Consider removing the override of `AllowShortLambdasOnASingleLine`.

Reply:
Trying out setting it to `Inline` (per https://clang.llvm.org/docs/ClangFormatStyleOptions.html#allowshortlambdasonasingleline)